docs/website-docs/record-n-upload.md

```diff
@@ -0,0 +1,125 @@
+Android
+---------
+
+## Record
+
+### How to start a trace recording
+The 'Record trace' button should be available in your quick settings. If it is not, go back to the section on [enabling trace recording](device-set-up.md#3-make-sure-tracing-is-enabled) and ensure you enabled __Show Quick Settings tile__.
+
+### 1. Install instrumented app
+
+Install the app that you want to analyze instrumented with PS plugin. Don't have the app? Learn [here](../android/gradle.md) how to build instrumented app.
+
+### 2. Select a user flow you want to analyze
+
+At Product Science, performance optimization starts with defining key user flows that bring the most value to users.
+
+##### What is a user flow?
```

Comment:
This is duplicated for iOS steps, should there be a separate file or included in glossary for user flow methodology?